crypto/storage/azure/interface.go

```diff
@@ -34,13 +34,24 @@ type Config struct {
 	// UseHSM specifies whether	to store the key in a hardware security module (HSM).
 	// If true, the Azure Key Vault must be configured for HSM usage.
 	UseHSM bool `koanf:"hsm"`
+	// Credential specifies the credential to use for authentication to the Azure Key Vault.
+	Credential CredentialConfig `koanf:"credential"`
```

Comment:
I also saw a `TokenCredential` interface somewhere, is that a more specific name that could be used to avoid confusion with verifiable credentials?

Reply:
I believe "credential" is the term they stick to that gives you access to an identity (user/account), through a token. 

```
TokenCredential represents a credential capable of providing an OAuth token.
```

I can change it to "Auth".